packages/origin-247-certificate/src/offchain-certificate/offchain-certificate.service.ts

```diff
@@ -217,6 +240,8 @@ export class OffChainCertificateService<T = null> {
                 });
                 await this.createAggregate(events);
             }
+
+            await transferBatchCommandValidator(commands);
```

Comment:
And here, before creating aggregate or anything, this should be first thing in the code

Reply:
done